docs/agents/ASCPHandlingEventThatInitiatesAgentSCPProgram.tex

```diff
@@ -0,0 +1,25 @@
+\begin{SCn}
+
+\scnheader{Агент обработки события в sc-памяти, инициирующего агентную scp-программу}
+\scnidtf{sc-агент обработки события в sc-памяти, инициирующего агентную scp-программу}
+\scntext{примечание}{Первичным условием инициирования агента обработки события в sc-памяти, инициирующего агентную scp-программу, является элементарное sc-событие.}
+\scntext{примечание}{Агент обработки события в sc-памяти, инициирующего агентную scp-программу, используется для осуществления логики scp-агентов: реагирование на первичное условие инициирования и запуск интерпретации scp-программы.}
```

Comment:
```suggestion
\scntext{примечание}{Агент обработки события в sc-памяти, инициирующего агентную scp-программу, используется для осуществления логики scp-агентов --- реагирования на первичное условие инициирования и запуск интерпретации scp-программы.}
```

Reply:
no, [see](https://ru.wikipedia.org/wiki/%D0%91%D0%B5%D1%81%D1%81%D0%BE%D1%8E%D0%B7%D0%BD%D0%BE%D0%B5_%D1%81%D0%BB%D0%BE%D0%B6%D0%BD%D0%BE%D0%B5_%D0%BF%D1%80%D0%B5%D0%B4%D0%BB%D0%BE%D0%B6%D0%B5%D0%BD%D0%B8%D0%B5#:~:text=1.%20%D0%92%D1%82%D0%BE%D1%80%D0%B0%D1%8F%20%D1%87%D0%B0%D1%81%D1%82%D1%8C%20%D1%80%D0%B0%D0%B7%D1%8A%D1%8F%D1%81%D0%BD%D1%8F%D0%B5%D1%82%2C%20%D1%80%D0%B0%D1%81%D0%BA%D1%80%D1%8B%D0%B2%D0%B0%D0%B5%D1%82%20%D1%81%D0%BE%D0%B4%D0%B5%D1%80%D0%B6%D0%B0%D0%BD%D0%B8%D0%B5%20%D0%BF%D0%B5%D1%80%D0%B2%D0%BE%D0%B9%20(%D0%BC%D0%BE%D0%B6%D0%BD%D0%BE%20%D0%B2%D1%81%D1%82%D0%B0%D0%B2%D0%B8%D1%82%D1%8C%20%D0%B0%20%D0%B8%D0%BC%D0%B5%D0%BD%D0%BD%D0%BE))